File: AppGenerateFiles/dbo/Model/Catalogo_Agentes.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Catalogo_Agentes : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_agente { get; set; }
       public string? identificacion { get; set; }
       public string? nombre { get; set; }
       public string? telefono { get; set; }
       public string? direccion { get; set; }
       public DateTime? fecha { get; set; }
       public int? Id_Tipo_Agente { get; set; }
       public string? Estado { get; set; }
       [ManyToOne(TableName = "Catalogo_Tipo_Agente", KeyColumn = "Id_Tipo_Agente", ForeignKeyColumn = "Id_Tipo_Agente")]
       public Catalogo_Tipo_Agente? Catalogo_Tipo_Agente { get; set; }
       [OneToMany(TableName = "Security_Users", KeyColumn = "id_agente", ForeignKeyColumn = "id_agente")]
       public List<Security_Users>? Security_Users { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Catalogo_Agentes.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Tipo_Agente }  from './Catalogo_Tipo_Agente.js'
import { Security_Users }  from './Security_Users.js'
class Catalogo_Agentes extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_agente;
   /**@type {String}*/ identificacion;
   /**@type {String}*/ nombre;
   /**@type {String}*/ telefono;
   /**@type {String}*/ direccion;
   /**@type {Date}*/ fecha;
   /**@type {String}*/ Estado;
   /**@type {Catalogo_Tipo_Agente} ManyToOne*/ Catalogo_Tipo_Agente;
   /**@type {Array<Security_Users>} OneToMany*/ Security_Users;
}
export { Catalogo_Agentes }
